api/AudioScheduledSourceNode.json

```diff
@@ -71,10 +71,10 @@
             }
           ],
           "safari": {
-            "version_added": null
+            "version_added": false
```

Comment:
The IDL file was added in https://trac.webkit.org/changeset/264538/webkit so soon this will be supported. But for now, this is correct. Per https://github.com/mdn/browser-compat-data/issues/6656 it doesn't really matter, but we can sort it out there.